File: hw16/sqrts.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

int main(int argc, char **argv)
{
	int max = atoi (argv[1]);
	int n;
  	printf("sqrt(n)\n--------\n");
  	for (n=0;  n<=max;  n++)
    	printf("%f\n", sqrt(n));
  	return 0;
}

File: hw22/Sqrts.java
import java.io.*;

class Sqrts {
  public static void main(String[] args) {
    System.out.println("sqrt(n)");
    System.out.println("--------");
    int max = Integer.parseInt(args[0]);
    int n;
    for (n=0;  n<=max;  n++)
      System.out.println(Math.sqrt(n));
    return;

  }
}